.storybook/loaders/icon-loader.js

```diff
@@ -0,0 +1,40 @@
+export const IconLoader = async () => ({
```

Comment:
One minor suggestion would be to add a comment about what this function does. Not a holdup for moving this through, as this was just moving it to its own file.

It loads every SVG from both icons package and stores the raw SVG contents as a string in the object? Interesting, I didn't know we were loading them all at once.